framework/sandstone.cpp

```diff
@@ -2339,13 +2339,15 @@ run_one_test(const test_cfg_info &test_cfg, SandstoneApplication::PerCpuFailures
     sApp->current_test_duration = test_duration(test_cfg);
     first_iteration_target = MonotonicTimePoint::clock::now() + 10ms;
 
-    if (sApp->max_test_loop_count) {
-        sApp->shmem->current_max_loop_count = sApp->max_test_loop_count;
+    if (test->fracture_loop_count) {
+        sApp->shmem->current_max_loop_count = test->fracture_loop_count;
```

Comment:
The fallback `else` at the end of this construct does not make sense anymore. However, the question is why we're honoring `fracture_loop_count` over `max_test_loop_count` with this change.

Reply:
ok. @mcisowsk please include the reasoning in the commit message (and, optionally, in the PR description).